### DISCUSSION TITLE
Determining underlying device

### DISCUSSION THREAD

**Discussion by TobiasHT5** [flet-dev/flet#1462] [Q&A]
Is there any way for flet to tell on which device it's running on? I would like to build a different experience for mobile and desktop web applications but I need to first determine if the app is running on mobile or desktop.

**Comment by SKbarbon** [comment]
@ht-thomas there is a way which is to check if the page.width is smaller than 490 then do a custom ui then, and if it’s bigger then make another desktop ui thing.

I think this method is better because its more fit for even deaktop on small windows.

**Comment by ndonkoHenri** [comment]
- [page.platform](https://flet.dev/docs/controls/page#platform)
- [page.pwa](https://flet.dev/docs/controls/page#pwa)
- [page.web](https://flet.dev/docs/controls/page#web)
